src/coreclr/src/System.Private.CoreLib/src/System/Runtime/InteropServices/ComEventsInfo.cs

```diff
@@ -21,6 +23,7 @@ private ComEventsInfo(object rcw)
 
         public static ComEventsInfo? Find(object rcw)
         {
+            Debug.Assert(OperatingSystem.IsWindows());
```

Comment:
Since this is COM, would it be more appropriate to mark the whole class as `[SupportedOSPlatform("windows")]`?

Reply:
Yes if "COM" is windows only that is the most appropriate way